traitsui/qt4/video_editor.py

```diff
@@ -314,6 +323,9 @@ def _media_status_changed_emitted(self, error):
     def _buffer_status_changed_emitted(self, error):
         self.buffer = self.media_player.bufferStatus()
 
+    def _notify_interval_changed_emitted(self, interval):
+        self.notify_interval = interval / 1000.0
```

Comment:
Does this need to be in a `with self.updating_value():` block like the `position` trait?

Reply:
Possibly.... I couldn't understand why this was done for `_position_changed_emitted` but not the other `*_emitted` methods though. Is there a reason?

Reply:
That was the fix for choppy playback on Linux and Windows. See https://github.com/enthought/traitsui/pull/745#discussion_r607888802 for more discussion

Basically, we want to call `self.media_player.setNotifyInterval` if the `notify_interval` trait changed for any other reason **except** `_notify_interval_changed_emitted` being called.

Unless it doesn't matter.

Reply:
Ah that makes sense. It doesn't seem to matter on Mac/Windows. (Added an `observe` for `notify_interval` and Qt never emits a change signal, when both streaming and watching a local video.)